elegant-emacs-dark-theme.el

```diff
@@ -0,0 +1,628 @@
+;;; elegant-emacs-light-theme.el --- A very minimal but elegant and consistent theme
```

Comment:
If it is to be published on Melpa, is the name "elegant-emacs-..." the right name or can we used something like "elegance-light-theme..." directly (if not already used) ?

Reply:
we can use whatever name we want as long as all the places where we specify it are equal. `elegance ... theme` is not used but I worry people are already familiar with the name elegant-emacs. We could open an issue and link it on reddit to see what people think

Reply:
ok, let's keep elegant-emacs then.